src/targets/gpu/compile_hip.cpp

```diff
@@ -166,6 +166,7 @@ struct hiprtc_program
                        options.end(),
                        std::back_inserter(c_options),
                        [](const std::string& s) { return s.c_str(); });
+        c_options.push_back("-I/opt/rocm/include");
```

Comment:
This include should not be added. When using hiprtc we should never be including files on the filesystem.

If this is to fix the include of `hip/hip_runtime.h` in the unit tests [here](https://github.com/ROCm/AMDMIGraphX/blob/develop/test/gpu/jit.cpp#L42) then the unit tests need to be updated to only include hip_runtime.h when not compiling with hiprtc as hiprtc already provides this header by default.